src/isar/config/settings.py

```diff
@@ -213,6 +214,32 @@ class Settings(BaseSettings):
         ]
     )
 
+    # Available decks for the robot with default localization pose
+    AVAILABLE_DECKS: List[AssetDeck] = Field(
+        default=[
+            AssetDeck(
+                deck_name="Placeholder deck 1",
+                default_pose=Pose(
+                    position=Position(x=1, y=1, z=1, frame=Frame(name="asset")),
+                    orientation=Orientation(
+                        x=0, y=0, z=0, w=1, frame=Frame(name="asset")
+                    ),
+                    frame=Frame(name="asset"),
+                ),
+            ),
+            AssetDeck(
+                deck_name="Placeholder deck 2",
+                default_pose=Pose(
+                    position=Position(x=2, y=2, z=2, frame=Frame(name="asset")),
+                    orientation=Orientation(
+                        x=0, y=0, z=0, w=1, frame=Frame(name="asset")
+                    ),
+                    frame=Frame(name="asset"),
+                ),
+            ),
+        ]
+    )
+
```

Comment:
I don't know the best practise here but maybe the env variable should be a json that is constructed to an object in code?